rest-api-spec/src/main/resources/rest-api-spec/api/cat.thread_pool.json

```diff
@@ -36,7 +36,6 @@
         "type":"enum",
         "description":"The multiplier in which to display values",
         "options":[
-          "",
```

Comment:
I don't think this should be removed in 7.x. In removing this, it may introduce a breaking change to the client's API and because the major versioning of the clients is in lock step with Elasticsearch, it would be a breaking change to the client's API within a major version.

To move forward, we could special case handle an empty string enum option on `size` for `cat.threadpool` in the JSON schema.

Reply:
If it were removed, clients where it would introduce a breaking change would need to specially handle this in their generator

Reply:
Hmm... i don't see a good way to special case this in the JSON schema. However, if there is a good way I am still a bit hesitant to special case this in the schema. Splippery slope argument about too many special cases and a schema stops being a schema due to special cases ... and a real world concern of readability and keeping 7.x closer inline with master.

I think that since there exists an API with a `""` and there is support for it, that `""` should validate with the schema (but only for 7.x). I added schema support across the board for `""` in https://github.com/elastic/elasticsearch/pull/56736/commits/af2780a5d74d460806b89452677977bd4cbd053e. This will not be forward ported to master, and since all changes generally go through master first it is pretty unlikely that someone will accidentally try to support `""` since on master it will fail validation. 

I understand the desire to not break the clients, so I think we can either 1) support `""` for 7.x, or 2) hard code to ignore this file in the task itself (like we do with _common), or 3) figure out how to special case this in the schema (I will need some assistance with that).   

My preference is 1 (as in the latest update to this PR) (then 2, then 3)  

thoughts?

Reply:
> since all changes generally go through master first it is pretty unlikely that someone will accidentally try to support "" since on master it will fail validation.

Good point; this sounds like a good argument for not needing to special case `""` to cat.threadpool in 7.x since any API changes would be backported from master which would disallow.